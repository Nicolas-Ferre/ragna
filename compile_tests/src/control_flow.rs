fn main() {}

#[ragna::gpu]
mod gpu {
    fn loops() {
        'outer: while true {}
    }
}
