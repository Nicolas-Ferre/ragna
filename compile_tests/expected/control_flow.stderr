error: labels not supported
 --> compile_tests/src/control_flow.rs:6:9
  |
6 |         'outer: while true {}
  |         ^^^^^^